rust/sqlite/src/db.rs

```diff
@@ -19,13 +19,19 @@ use tokio::io;
 pub struct SqliteDb {
     conn: SqlitePool,
     migration_hash_type: MigrationHash,
+    pub push_logs_batch_size: usize,
```

Comment:
I think this API design is incorrect sqlitedb has no knowledge of push_logs. This is hacky forcing this into here.